apps/src/authoredHints.js

```diff
@@ -183,19 +183,11 @@ AuthoredHints.prototype.updateLightbulbDisplay_ = function (animate) {
     this.promptIcon.parentNode.insertBefore(this.lightbulb, this.promptIcon);
   }
 
-  // If there are more than nine hints, simply display "9+"
-  var hintText = (hintCount > 9) ? "9+" : hintCount;
-  if (hintCount === 0) {
-    this.lightbulb.innerHTML = lightbulbDimSVG;
-  } else {
-    this.lightbulb.innerHTML = lightbulbSVG;
-    this.lightbulb.querySelector('#hintCount').textContent = hintText;
-  }
-
-  var bulb = document.getElementById("bulb");
-  if (animate && bulb) {
-    bulb.setAttribute('class', 'animate-hint');
-  }
+  React.render(React.createElement(Lightbulb, {
+    count: hintCount,
+    lit: hintCount > 0,
```

Comment:
Could the lit logic just live inside Lightbulb (i.e. we also have it be dependent on count) or are there times when we'll want the caller to be able have it be lit independent of count?